mutacc/utils/bam_handler.py

```diff
@@ -130,30 +153,36 @@ def out_file(self):
 
         return str(self.out_name)
 
-    #This method may be implemented using picard FilterSamReads or another faster command line
```

Comment:
Have you tried bedtools or samtools?
1. bedtools intersect -abam file.bam -b filter.bed -v > filtered.bam
2. samtools view -U -L filter.bed > filtered.bam

Reply:
That might be a better option. The only problem is that we might want to filter out all the mates to the reads overlapping the regions as well.